contributors/devel/development.md

```diff
@@ -150,9 +151,8 @@ images.
 
 - The image for cross compiling in [build/build-image/cross].
   The `VERSION` file and `Dockerfile`.
-- Update the desired Go version in Dockerfile for the [e2e][e2e-image] and [test][test-image].
-  This requires pushing the [e2e][e2e-image] and [test][test-image] images that are `FROM` the desired Go version.
```

Comment:
The test image in `test/images/Makefile` needs to be updated too
https://github.com/kubernetes/kubernetes/pull/74632/files

Reply:
Thank you for the ctach @cblecker !
Fixed it